static/auth-result/authFailure/index.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title></title>
+    <link rel="stylesheet" href="../styled.css" />
+  </head>
+  <body>
+    <container id="authFailure">
+      <content>
+        <img src="ErrorIcon.svg" alt="Error" />
+        <heading>Something went wrong</heading>
+        <text
+          >We could not grant access to your account. Please try again.</text
+        >
+      </content>
+    </container>
+    <script>
+      if (window.opener) {
+        const params = new URLSearchParams(window.location.search);
+
+        if (params.has('success')) {
```

Comment:
@akostevich-atlassian do we have need this check anymore?

Reply:
No, we can safely remove this check since this page gets specific to the "success" case.